src/Exporter.php

```diff
@@ -47,13 +53,26 @@ public function crawl(bool $crawl): self
         return $this;
     }
 
-    public function paths(array $paths): self
+    public function paths(...$paths): self
     {
+        $paths = is_array($paths[0]) ? $paths[0] : $paths;
+
         $this->paths = array_merge($this->paths, $paths);
```

Comment:
Maybe shorten it
```php
$this->paths = array_merge($this->paths, Arr::wrap($paths));
```

Reply:
`Arr::wrap()` does something different. `is_array($paths[0]) ? $paths[0] : $paths` takes the first argument as array or all arguments as array.
```php
paths(['path1', 'path2']);

paths('path1', 'path2');
```